src/Opdex.Platform.Application/Assemblers/MarketTokenDtoAssembler.cs

```diff
@@ -24,13 +24,6 @@ public async Task<MarketTokenDto> Assemble(MarketToken token)
     {
         var marketTokenDto = _mapper.Map<MarketTokenDto>(token);
 
-        var summary = await _mediator.Send(new RetrieveTokenSummaryByMarketAndTokenIdQuery(token.Market.Id, token.Id, findOrThrow: false));
-
-        if (summary != null)
-        {
-            marketTokenDto.Summary = _mapper.Map<TokenSummaryDto>(summary);
-        }
-
```

Comment:
I think this is still necessary for getting singular market tokens. so maybe wrap in `if(marketTokenDto.Summary == null)`

Reply:
Good point, I'd missed this. Deciding whether or not to add back this call or get the token summary within the query. Potentially we have a flag on the select token query for whether to load related entities (the token summary), then we avoid any performance penalties where the summary isn't needed. Although for now will keep it simple and add back the retrieval call in the assembler

Reply:
Thinking retrieving the summary from within the select handler is better. Since now we have `GET /markets/{market}/tokens{token}`, `GET /markets/{market}/tokens` and `GET /tokens` returning the summary, yet `GET/tokens/{token}` is not. For consistency and since we can, maybe we should.